src/script/script_info.cpp

```diff
@@ -163,23 +172,22 @@ SQInteger ScriptInfo::AddSetting(HSQUIRRELVM vm)
 	/* Check if default_value is set. Although required, this was changed with
 	 * 14.0, and as such, older AIs don't use it yet. So we convert the older
 	 * values into a default_value. */
-	if ((items & 0x080) == 0) {
+	if (!present.Test(ScriptConfigItemKey::DefaultValue)) {
 		/* Easy/medium/hard should all three be defined. */
```

Comment:
Outdated comment, but I guess useful for historical documentation of the old difficulty-specific defaults.